### DISCUSSION TITLE
エディタ内で Note 記法のサジェスト表示

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#292] [要望・アイデア・不具合]
Note 記法は `info`、`warn`、`alert` の 3 つがあります。エディタ内で `:::note` をタイプした時点で、これらのサジェスト（と補完）があると分かりやすいと思います。

個人的な話ですが、`warn` を `warning` に間違えたり、`alert` を忘れてしまったりするため、サジェストがあると間違えることがなくなり助かります。

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見ありがとうございます🙏こちらできると便利そうです！
前向きに検討させていただきますので、実装までお待ちいただけますと幸いです🙇

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告が遅くなり申し訳ありません🙇

`:::note` をタイプした時ではないのですが、「/」コマンドによる補完機能でノート記法のサジェスト機能を実装いたしました！
https://qiita.com/release-notes/2023-10-18

ぜひご活用いただけると幸いです✨

**Reply by yarakos95** [reply]
リリースノートから `/` から Note 記法のサジェストが表示されることは把握していました 🎉 
実装ありがとうございます！

すっかりこの提案を挙げていたことも忘れていました… 😓

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi 
返信いただきありがとうございます！
リリースノートで確認いただいていたとのこと、よかったです🙌✨

改善にお時間をいただく状況となっており恐れ入りますが、引き続き改善を続けていきますので、これからもよろしくお願いします🙇
